time.qmd

```diff
@@ -112,3 +112,61 @@ plot(t, run(3.8, sys2, t), ylim = 0:1, type = "l",
 ```
 
 None of these dynamics appeared in the continuous time version!
+
+## Incidence calculations
```

Comment:
I wonder if we go with "Variables that reset" or something - of which incidence is a special case?

Reply:
Changed